stdlib/public/core/CompilerProtocols.swift

```diff
@@ -350,8 +350,8 @@ public protocol _ExpressibleByBuiltinBooleanLiteral {
 /// A type that can be initialized with the Boolean literals `true` and
 /// `false`.
 ///
-/// Only three types provided by Swift---`Bool`, `DarwinBoolean`, and
-/// `ObjCBool`---are treated as Boolean values. Expanding this set to include
+/// Only four types provided by Swift---`Bool`, `DarwinBoolean`, `ObjCBool`, and
```

Comment:
I wonder if this shouldn't be "Only" at this point…

Reply:
haha, happy to un-only this :-)